src/rad/buffer.clj

```diff
@@ -46,6 +75,10 @@
                    the-whole-new-line (into above-current-plus-rest-of-current every-line-below-current-line)]
                the-whole-new-line)
 
+    :backspace (delete-char-backwards @current-buffer point)
```

Comment:
This should be the char `\backspace`, so that it's not dependent upon my terminal implementation